docs/03_server/12_tooling/02_intellij-plugin.mdx

```diff
@@ -119,125 +199,208 @@ To clean up the UI, use the **Services** window for better organisation (only av
-Once started, this launches a browser tab showing your application's login screen.
+### OpenAPI
 
-![Debug Window](/img/intellij-ui.png)
+The plugin can contact the router to provide [an openapi spec](/server/integration/open-api).
+To do this; the plugin will ask for a username and password to log in to your application.
+It will then request the openapi spec, and show a dialog to save it into your project.
 
-:::info
-Note that we are using [GENESIS_ROUTER](../../../server/configuring-runtime/genesis-router/) as our API_HOST; its default port is 9064. Therefore, your router port in the settings should be as below.
+To start this process, click on the Genesis menu and then on "Generate OpenApi Spec":
 
-![Router Settings](/img/plugin-router.png)
-:::
+![Open Api in menu](/img/intellij-plugin/menu-openapi.png)
+
+This will then trigger a username and password dialog.
+
+![Open Api Dialog](/img/intellij-plugin/openapi-dialog.png)
+
+Afterwards, you will be asked to save the spec, and it should open automatically.
+
+### Using the Terminal
+
+The Genesis Plugin integrates directly into IntelliJ's terminal.
+Most commands supported on the server can be run straight from there.
+Supported commands are highlighted in green,
+and can be run using CTRL+Enter on Windows, or Command+Enter on MAC.
 
-### Making a change
+For example:
 
-When changing files related to a certain process, there is no need to redeploy the entire application.
+![Using the Terminal](/img/intellij-plugin/terminal-support.png)
 
-When you restart a process, the modules relating to the process will be rebuilt and re-deployed automatically.
+Or:
 
-For wider-reaching changes, such as changing dependencies, you must redeploy the application:
+![Using SendIt](/img/intellij-plugin/terminal-sendit.png)
 
-1. Click on the **Deploy Genesis** button on the toolbar.
+#### Useful Commands
 
-![Deploy](/img/intellij-deploy1.png)
+These are some of the commonly used commands in the plugin:
 
-2. Redeploying the application requires the Genesis processes to be stopped. When you are prompted for this, click **ok** to continue.
+* [DbMon](/operations/commands/dbmon/) - the Genesis Database client
+* [DumpIt](/operations/commands/server-commands/#dumpit) - export CSV data into the database
+* [SendIt](/operations/commands/server-commands/#sendit) - import CSV data from the database
+* [LogLevel](/operations/commands/server-commands/#loglevel) - dynamically adjust log levels of running processes
 
-![Deploy Prompt](/img/intellij-deploy2.png)
+For a full list of Genesis commands, please see [here](/operations/commands/server-commands/).
 
-This starts the build processes and the logs will be shown below.
+### Resource Daemon
+
+The resource daemon provides information about locally running processes.
+It is what drives the information shown both in the mon window in the plugin, and the mon command on the server.
+The plugin can automatically start the daemon when needed.
```

Comment:
Should mention that regardless of the setting the daemon will start if you start a process and it isn't running already